src/test/spec/client_side_encryption.rs

```diff
@@ -25,11 +26,11 @@ async fn run_unified() {
 async fn run_legacy() {
     let _guard: RwLockWriteGuard<()> = LOCK.run_exclusively().await;
 
-    // TODO RUST-528: Unskip timeoutMS.json when CSOT is implemented.
-    #[cfg(not(feature = "openssl-tls"))]
-    let skipped_files = &["timeoutMS.json", "kmipKMS.json"];
-    #[cfg(feature = "openssl-tls")]
-    let skipped_files = &["timeoutMS.json"];
+    // TODO RUST-528: unskip this file
+    let mut skipped_files = vec!["timeoutMS.json"];
+    if cfg!(not(feature = "openssl-tls")) {
```

Comment:
using the runtime `cfg!` macro rather than conditional compilation allows us to avoid unused mut warnings